gstd/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "gstd"
 description = "Gear programs standard library"
-keywords = ["gear", "std", "no-std", "wasm", "smart-contracts"]
+keywords = ["gear", "std", "no-std", "wasm", "programs"]
```

Comment:
I think `"programs"` doesn't make sense in this context

Reply:
Agree, reverted in 7bde400